packages/react/src/Link/Link.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render } from '@testing-library/react'
+import { Link } from './Link'
+import '@testing-library/jest-dom'
+
+describe('Link', () => {
+  const linktext = 'Linktext'
+  it('renders with href attribute', () => {
+    const { container } = render(<Link href="#">{linktext}</Link>)
+    const link = container.querySelector('a:only-child')
+    expect(link).toBeInTheDocument()
+    expect(link).toHaveAttribute('href', '#')
+  })
+
+  it('renders with target attribute', () => {
+    const { container } = render(
+      <Link href="#" target="_blank">
+        {linktext}
+      </Link>,
+    )
+    const link = container.querySelector('a:only-child')
+    expect(link).toBeInTheDocument()
+    expect(link).toHaveAttribute('target', '_blank')
+  })
+
+  it('renders with rel attribute', () => {
+    const { container } = render(
+      <Link href="#" rel="noopener">
+        {linktext}
+      </Link>,
+    )
+    const link = container.querySelector('a:only-child')
+    expect(link).toBeInTheDocument()
+    expect(link).toHaveAttribute('rel', 'noopener')
+  })
```

Comment:
Do we need to test this? These are standard html anchor attributes. Maybe we only need to test if the `Link` component renders an anchor